core/common/src/main/java/alluxio/ConfigurationDocGenerator.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
```

Comment:
Can you move this file to dir `shell/src/main/java/alluxio/cli/`? (as well as its unit test file to `shell/src/test/java/alluxio/cli/`.
The reason is we put these tools / shell commands in module `shell`; whereas classes in `core/common` module will be included in all jars which is unnecessary